src/test/jtx/Env.h

```diff
@@ -514,7 +536,7 @@ class Env
     /** Gets the TER result and `didApply` flag from a RPC Json result object.
      */
     static ParsedResult
-    parseResult(Json::Value const& jr);
+    parseResult(Json::Value const& jr, bool checkTER);
```

Comment:
I don't quite understand what this parameter is supposed to be doing. Can we add an explanation in the docstring?

Reply:
A few more checks, and I was able to get rid of `checkTER` entirely. The intention was to flag whether the request was related to submitting a transaction, and thus a `TER` would be expected in the result. There is no `TER` in the result if there isn't a transaction.